chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedPlateChartsUI.java

```diff
@@ -271,6 +273,18 @@ private void updateChart() {
 		}
 	}
 
+	private String getLabel(IWell well) {
+
+		LabelSetting referenceLabel = LabelSetting.valueOf(preferenceStore.getString(PreferenceConstants.P_PCR_REFERENCE_LABEL));
+		if(referenceLabel.equals(LabelSetting.SAMPLENAME))
```

Comment:
Please add curly braces. Are the formatting settings set in your IDE?

Reply:
I enabled the `Use blocks in if/while/for/do statements` save action now.